### PR TITLE
#173 new edit profile

### PR DESCRIPTION
**Pull request by leroy-p**

### CODE REVIEW

File: webapp/app/components/EditGroup/index.js [hunk 1]
```diff
@@ -71,9 +72,20 @@ export class EditGroup extends React.Component {
...
   handleClick(event) {
-    var groups = [];
-    groups.push(this.group);
-    this.props.editGroupRequest(groups);
+    var group = {};
+    var users = [];
+    this.group.users.map(function(user) {
+      users.push(user.id);
+    });
+    console.log('CUCUCUCUCUCUCU');
```
Comment (michau-k) [inline webapp/app/components/EditGroup/index.js]: Remove console.log

File: webapp/app/components/EditUser/indexOld.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+import React from 'react';
+import { FormGroup, FormControl, ControlLabel, PageHeader, Col } from 'react-bootstrap';
+import { LinkContainerButton } from '../Button';
+import RadioGroup from '../RadioGroup';
+import styles from './styles.css';
+
+export class EditUser extends React.Component {
+  constructor(props) {
+    super(props);
+    this.user = {};
```
Comment (michau-k) [inline webapp/app/components/EditUser/indexOld.js:14]: Should be in redux, or at least in the component state, same for line below

File: webapp/app/containers/CreateUser/actions.js [hunk 3]
```diff
@@ -25,19 +25,16 @@ export function createUser(user) {
...
       .type('json')
       .send({ users })
       .end((err, res) => {
-        console.log('reponse a /api/logged-in/admin/users/create :');
-        console.log(res.body);
         dispatch(createUser(res.body));
         if (!res.body.error) {
-          dispatch(push('/edit-user/' + users[0].name));
+          var last = res.body.users.length - 1;
+          dispatch(push('/edit-user/' + res.body.users[last].id));
```
Comment (michau-k) [inline webapp/app/containers/CreateUser/actions.js]: Dispatch(push) ?

File: webapp/app/containers/CreateGroup/actions.js [hunk 4]
```diff
@@ -25,19 +25,16 @@ export function createGroup(group) {
...
       .type('json')
       .send({ groups })
       .end((err, res) => {
-        console.log('reponse a /api/logged-in/admin/groups/create :');
-        console.log(res.body);
         dispatch(createGroup(res.body));
         if (!res.body.error) {
-          dispatch(push('/edit-group/' + groups[0].name));
+          var last = res.body.groups.length - 1;
+          dispatch(push('/edit-group/' + res.body.groups[last].id));
```
Comment (michau-k) [inline webapp/app/containers/CreateGroup/actions.js]: Dispatch(push) ?

**Comment by BernardJeremy** [comment]
fix #174 too

**Pull request merged**